src/lib/utils/deps.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Installs Tensorflow
+ */
+export const installTF = () => {
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
This is to disable `tfjs-node` during Jest test. Please read more at https://github.com/tensorflow/tfjs/issues/545